### DISCUSSION TITLE
Update properties after initialization

### DISCUSSION THREAD

**Discussion by pvictor** [sa-si-dev/virtual-select#124] [Q&A] [answered]
Is it possible to update properties after initializing virtual select component with `VirtualSelect.init` ?

My immediate use-case will be to update text properties according to an i18n variable.

Thanks!

**Comment by sa-si-dev** [comment, marked as answer]
As of now, we could set options, value, and disabled options after initialisation.

My suggestion for your use case is, call init method after getting i18n value.

**Comment by pvictor** [comment]
Thank you, yes and that's already great.

I want to be able to switch languages in my app. so I will destroy instances and re-initialize them I think.

Thank again!
